docs/_glossary/CommandInputBox.md

```diff
@@ -5,4 +5,4 @@ show-in: [ug, dg]
 
 The region located at the top-left of FoodRem's main window.
 
-To view more information, refer to the [Layout](/UserGuide.html#layout) section of the User Guide.
+To view more information, refer to the [Layout](#layout) section of the User Guide.
```

Comment:
I think this line breaks for the DG, seeing how the pipeline is failing as well. This section is supposed to show in the DG and link to the UG's layout section so just `[Layout](#layout)` doesn't seem to be working for the DG tho it does fix the broken link for UG.

Alternatively, since the actual link we want is `https://ay2223s1-cs2103t-w16-2.github.io/tp/UserGuide.html#layout` while the previous link links to `https://ay2223s1-cs2103t-w16-2.github.io/UserGuide.html#layout`, maybe just adding the missing `/tp` like `[Layout](/tp/UserGuide.html#layout)` would suffice.